plugin/test/sample-test.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { graphql } from "msw";
+import { SourceNodesArgs } from "gatsby";
+import { setupServer } from "msw/node";
+import { createOperations } from "../src/operations";
+
+const server = setupServer(
+  graphql.query("OPERATION_BY_ID", (req, res, ctx) => {
+    return res(
+      ctx.data({
+        node: {
+          id: req.body?.variables?.id,
+          status: `COMPLETED`,
+        },
+      })
```

Comment:
Thank you @wardpeet for this hot tip, this is a really nice DX for mocking GraphQL requests!